### ISSUE TITLE
The Remote api failed to respond

### ISSUE THREAD

**Issue by sam0hack** [popcorntime/popcorntime#859]
Operating System Version:

Linux - Fedora 28

Popcorn Time Version:

0.3.10

Download date:

30 OCT 2018
#### Expected Behaviour

...

#### Actual Behaviour

...

#### Steps to reproduce the behavior

1. ... Start Desktop and click on Series menu and it gives error The Remote api failed to respond check http://status.popcorntime.sh/  However, there is status Ok Also app cannot load any movies series etc.

**Comment by Persei08** [comment]
Sound like you can't reach the API, maybe your ISP is blocking PT API domain. Switch your DNS for Google DNS (8.8.8.8, 8.8.4.4) or Cloudflare DNS (1.0.0.1, 1.1.1.1) and see if it work better

**Comment by besoeasy** [comment]
why don't use shufflix api ?

https://github.com/besoeasy/shufflix

**Comment by impsdc** [comment]
Still 
Operating System Version:

Linux - Pop_os 20

Popcorn Time Version:

0.4.4

Can not understand what I have to do with the switching dns thing and the link is not longer available

**Comment by dutchy1001** [comment]
This issue is very old. It would be better to make a new issue.
If you haven't updated your app then I suggest to do so from **[here](https://ci.popcorntime.app/job/Popcorn-Time-Desktop/lastStableBuild/)**. Report back (in this case make a new issue) if the issue still exist.

**Comment by impsdc** [comment]
Working now, there are so many ways to download this app and they all saying it was the last versions. 
Anyway, Thanks !

**Comment by sam0hack** [comment]
Well, the issue was ISP blocking the torrent URL's and with VPN it was working fine. However, If they add something for the status check to see whether a user able to access their torrent URL would save a ton of time debugging.

**Comment by dutchy1001** [comment]
@sam0hack 
why?? You can connect or you can't. That is a simple observation.
There is a status check that will tell you the availability of the torrent in the right top corner of the movie/series page, (green/yellow/red and gray dot) that also tells you the amount of seeds each has.

**Comment by sam0hack** [comment]
Actually, When isp blocking the URLs this App seems broken and that's why it is really hard to know what went wrong.

**Comment by dutchy1001** [comment]
if your asking to add a timer for "finding" content then you should make a "feature" request, the devs can accept this or disregard it. 
This would be a better option then answering in a nerco topic.

**Comment by sam0hack** [comment]
No, I don't need that "timer for finding content" my issue was this " _(Start Desktop and click on the Series menu and it gives an error The Remote api failed to respond check.)_  " So my feature request would be to make something that can check if application connected to the server. like we have on the phone which invoke message if phone not connected to the internet So something similar to this but with extra feature like popup a message if URL's are being blocked by ISP. I'll create a feature request or maybe I will add that in code.

**Comment by kiriles90** [comment]
@sam0hack Popcorn Time doesnt have a way to know if the reason you cant connect to the API urls is that they are blocked by your ISP or some other connection issue along the line. As for the occasions where your whole internet is down.. do you really need Popcorn Time to inform you of that? Can't you just look at your systray and find out with a glance?

**Comment by sam0hack** [comment]
Yeah, i tested that in the latest version. This is an old issue and when i created this issue there wasn't anything likr that. 
Anyway im happy with the latest version.

**Comment by kiriles90** [comment]
Yeah this was changed recently with #1942 (build 451)
